Workbooks/Virtual Machines - Network Dependencies/Connections Overview - Single VM/Connections Overview.workbook

```diff
@@ -96,1470 +167,1528 @@
       "name": "text - 2"
     },
     {
-      "type": 9,
+      "type": 12,
       "content": {
-        "version": "KqlParameterItem/1.0",
-        "crossComponentResources": [
-          "{WorkbookResource}"
-        ],
-        "parameters": [
+        "version": "NotebookGroup/1.0",
+        "groupType": "editable",
+        "items": [
           {
-            "id": "addcec31-b7ac-4715-a78d-9b803f86af8a",
-            "version": "KqlParameterItem/1.0",
-            "name": "TimeRange",
-            "type": 4,
-            "isRequired": true,
-            "value": {
-              "durationMs": 3600000
+            "type": 9,
+            "content": {
+              "version": "KqlParameterItem/1.0",
+              "crossComponentResources": [
+                "{WorkbookResource}"
+              ],
+              "parameters": [
+                {
+                  "id": "addcec31-b7ac-4715-a78d-9b803f86af8a",
+                  "version": "KqlParameterItem/1.0",
+                  "name": "TimeRange",
+                  "type": 4,
+                  "isRequired": true,
+                  "value": {
+                    "durationMs": 3600000
+                  },
+                  "typeSettings": {
+                    "selectableValues": [
+                      {
+                        "durationMs": 300000,
+                        "createdTime": "2019-01-28T23:37:45.024Z",
```

Comment:
did you manually copy this from somewhere else?  these time settings have a ton of fields that aren't actually used anymore, which makes me think this was copied+pasted?  the only fields required here or output anymore is `durationMs` which is adding a lot of bytes to this template for no reason?

Reply:
I didn't copy/paste this part. In fact, this section of the code is something I hadn't touched and is from before. I'm not sure why this is showing up in the diff.

Reply:
I guess this is showing up coz I added existing parameters into a group in order to conditional make groups visible.